Purchases.podspec

```diff
@@ -17,6 +17,8 @@ Pod::Spec.new do |s|
 
   s.ios.deployment_target = '9.0'
   s.osx.deployment_target = '10.12'
+  s.watchos.deployment_target = '6.2'
+  s.tvos.deployment_target = '9.0'
```

Comment:
This one differs with the one in the `Package.swift`

Reply:
this is the correct one, I'll update Package.swift